heartbeat/docker-compose

```diff
@@ -148,12 +141,19 @@ docker_compose_status()
 {
 	# use docker-compose ps if YML found, otherwise try docker ps and kill containers
 	if [ -r "$DIR/$YML" ]; then
-		STAT_MSG=$(cd $DIR ; $COMMAND ps)
-		LNWTH=$(echo "$STAT_MSG" | expand | head -n1 | wc -c)
-		STATEPOS=$(echo "$STAT_MSG" | expand | head -n1 | egrep -o 'State.*$' | wc -c)
-		OFFSET=$(($LNWTH-$STATEPOS+1))
-		PSNU=$(echo "$STAT_MSG" | grep "^${SVC}_" | wc -l)
-		UPNU=$(echo "$STAT_MSG" | grep "^${SVC}_" | expand | cut -c ${OFFSET}- | awk '{print $1}' | grep -w 'Up' | wc -l)
+
+                DKPS=$(cd $DIR; $COMMAND ps -q)
```

Comment:
You should use tabs for indentation to stay consistent with the rest of the agent.